src/lib/y2storage/proposal/lvm_creator.rb

```diff
@@ -51,23 +49,21 @@ def initialize(original_devicegraph)
       # Returns a copy of the original devicegraph in which the volume
       # group (and its logical volumes, if needed) have been created.
       #
-      # FIXME: the volume group will not be created if it does not contain
-      # any logical volume.
-      # FIXME: when reusing a volume group, all logical volumes will be
-      # removed in order to allocate new ones.
-      #
       # @param planned_vg [Planned::LvmVg]   Volume group to create
       # @param pv_partitions [Array<String>] names of the newly created
-      #     partitions that should be added as PVs to the volume group
+      #   partitions that should be added as PVs to the volume group
       # @return [Devicegraph] New devicegraph containing the planned volume group
       def create_volumes(planned_vg, pv_partitions = [])
         new_graph = original_devicegraph.duplicate
-        return new_graph if planned_vg.lvs.empty?
-
-        vg = planned_vg.reuse? ? find_vg(planned_vg, new_graph) : create_volume_group(new_graph)
+        vg =
+          if planned_vg.reuse?
+            find_vg(planned_vg, new_graph)
```

Comment:
Maybe I'm being nitpicking here. But this `if` reveals quite some inconsistencies in names and arguments order of the methods. `find_vg(planned_vg, graph)` vs `create_volume_group(graph, planned_vg)`.